### PR TITLE
Reflect feedback on document.domain article

### PR DESCRIPTION
**Pull request by agektmr**
Reflected feedback from @yoavweiss :

* I don't see a clear "how do I know my site is affected?" section. It seems worthwhile to add one about related console warnings, deprecation reports and [LightHouse deprecation audits](https://web.dev/deprecations/).
* Right now the article says that "Chrome plans to make document.domain immutable from Chrome 101". Would be good to soften this.
* Mozilla's position landed on "worth prototyping". It seems worthwhile to note we have their backing.
* I'd rephrase ["send ... header to continue.."](https://developer.chrome.com/blog/immutable-document-domain/#send-origin-agent-cluster-0-header-to-continue-using-documentdomain) as a last resort mechanism (so a "I can't make the proper change on time" solution, rather than a legitimate alternative).

### CODE REVIEW

File: site/en/blog/immutable-document-domain/index.md
Comment (yoavweiss) [inline site/en/blog/immutable-document-domain/index.md:209]: maybe "considers disabling `document.domain` by default worth prototyping"?

File: site/en/blog/immutable-document-domain/index.md
Comment (yoavweiss) [inline site/en/blog/immutable-document-domain/index.md:126]: s/by the Lighthouse/your site through LightHouse's deprecated API audit/

File: site/en/blog/immutable-document-domain/index.md
Comment (yoavweiss) [inline site/en/blog/immutable-document-domain/index.md:122]: Maybe "Affected websites will also get sent with deprecation reports, if they have a reporting endpoint set up. Learn more about [using the Reporting API](https://web.dev/reporting-api/#using-the-reporting-api) with either existing report collection services or by building your own in-house solution."

**Review by yoavweiss — COMMENTED** [review]
Looks good! A few comments and suggestions :)

**Review by yoavweiss — APPROVED** [review]
LGTM

**Review by heyawhite — APPROVED** [review]
LGTM  comments

I reviewed the Intent to Ship and it appears that the deprecation will be final in Chrome 105 (along with 3 LGTMs) -- if you think that's a fair change go ahead and make it. Otherwise, we can once again take back that language and say "in the future" with another blogpost when it's official.

File: site/en/blog/immutable-document-domain/index.md
Comment (agektmr) [reply site/en/blog/immutable-document-domain/index.md]: I've updated this section with the newer number: 106.

File: site/en/blog/immutable-document-domain/index.md
Comment (agektmr) [reply site/en/blog/immutable-document-domain/index.md]: I've updated this section with the newer number: 106.

**Comment by agektmr** [comment]
I've updated the article with the newer number: 106. Also, added a new "Updates" section at the top to clarify that we have updated the article.

Remaining task is to replace the DevTools Issues panel image with the latest string.

**Comment by agektmr** [comment]
@heyawhite [The string in the screenshot I embedded in the article](https://deploy-preview-2074--developer-chrome-com.netlify.app/blog/immutable-document-domain#how-do-i-know-if-my-site-is-affected) shows the old timeline, but I'd like to update this article ASAP because we've added a section about `document.domain` in [the 99 beta article](https://blog.chromium.org/2022/02/chrome-99-css-cascade-layers-new-picker.html) by mistake (it's already removed) and some people may find this article looking for information.

If you agree, could you remove `DO NOT MERGE` label and merge this when you see this message? Thanks!

**Comment by agektmr** [comment]
I've decided to publish this change though I'm aware the warning string in issues panel on the screenshot still shows old timeline. Let's fix it later.

**Comment by heyawhite** [comment]
@agektmr thanks for getting it merged -- we can fix it later this week

**Comment by agektmr** [comment]
@heyawhite thanks!

I found [an issue](https://bugs.chromium.org/p/chromium/issues/detail?id=1296013) that needs to be fixed before we can proceed. As soon as the fix lands, I will take a screenshot and replace it with the old one.

**Pull request merged**
